tensorboard/plugins/hparams/tf_hparams_table_view/tf-hparams-table-view.html

```diff
@@ -108,6 +108,7 @@
       .table-header {
         /* line-break overflowing column headers */
         white-space: normal;
+        word-break: break-word;
```

Comment:
Note that this does not work in Firefox: https://developer.mozilla.org/en-US/docs/Web/CSS/word-break
Might want to give `text-overflow: break-word` a try instead.

Some screen shot definitely would be nice too.